ui/src/components/Views/Configs/MountPointForm.tsx

```diff
@@ -12,12 +11,12 @@ import {
   Typography,
 } from '@mui/material';
 import React, { ReactElement, useEffect, useState } from 'react';
-import { useDDClient, useMountPoint } from '../../services/hooks';
-import { DockerImage } from '../../types';
-import { DownloadProgress } from '../DownloadProgress/DownloadProgress';
-
+import { LATEST_IMAGE } from '../../../constants';
+import { useDDClient, useMountPoint } from '../../../services';
+import { DockerImage } from '../../../types';
+import { DownloadProgress } from '../../Feedback';
 
-export const OnBoarding = (): ReactElement => {
+export const MountPointForm = (): ReactElement => {
   const { setMountPointUser } = useMountPoint();
   const ddClient = useDDClient();
   const [userState, setUserState] = useState({ loading: false, selectedUser: '', users: [] });
```

Comment:
conventionally we should start with states resulting of `useState` and then use other hooks